underscore.js

```diff
@@ -94,6 +93,30 @@
     return cb(value, context, Infinity);
   };
 
+  // Similar to ES6's rest param (http://ariya.ofilabs.com/2013/03/es6-and-rest-parameter.html)
+  // This accumulates the arguments passed into an array, after a given index.
+  var restArgs = function(func, startIndex) {
+    startIndex = startIndex == null ? func.length - 1 : +startIndex;
```

Comment:
Using function.length means functions bound with the `_.bind` fallback will work inconsistently with ones bound with `Function.prototype.bind` btw. Is this acceptable? @megawac @jridgewell 

```js
var func = function(a,b,c,d,e,f,g) {};

func.length; // 7
func.bind(null).length; // 7

Function.prototype.bind = null;
_.bind(func, null).length; // 0
```